cabal-install/src/Distribution/Client/ProjectBuilding.hs

```diff
@@ -1313,10 +1316,24 @@ buildInplaceUnpackedPackage verbosity
               Tar.createTarGzFile dest docDir name
               notice verbosity $ "Documentation tarball created: " ++ dest
 
+        -- Build info phase
+        {- buildInfo <- -}
+        whenBuildInfo $ do
+          -- Write the json to a temporary file to read it, since stdout can get
+          -- cluttered
+          let dir = distProjectCacheDirectory </> "buildinfo"
+          let fp = dir </> (unUnitId $ elabUnitId pkg) <.> "json"
```

Comment:
Filenames look like this: `hie-bios-0.8.0-inplace-hie-bios.json`
Not really stable and not nice to look at. Besides that these files are not strictly necessary, would there be a way to improve them?

Reply:
Is this subject to the magically disappearing vowel problem on Mac?

Reply:
Problem? I thought that is a feature? But probably no.
However, I am thinking about removing it and maybe we can talk about this in a follow-up pr.